### PR TITLE
Fix windows build

### PR DESCRIPTION
**Pull request by xobs**
This adds some patches to get `verilated-rs` running on Windows:

```powershell
[11:10:21 PM] D:/Code/verilated-rs> cargo run
   Compiling verilated v0.1.1 (D:\Code\verilated-rs\verilated)
   Compiling example v0.0.0 (D:\Code\verilated-rs\example)
    Finished dev [unoptimized + debuginfo] target(s) in 12.01s
     Running `target\debug\example.exe`
0: count_o = 0
1: count_o = 0
2: count_o = 1
3: count_o = 2
4: count_o = 3
5: count_o = 4
6: count_o = 5
7: count_o = 6
8: count_o = 7
9: count_o = 8
10: count_o = 9
11: count_o = 10
[11:10:34 PM] D:/Code/verilated-rs>
```

### CODE REVIEW

File: verilated/src/verilated_shim.cpp [hunk 1]
```diff
@@ -104,11 +104,6 @@ verilated_command_args(int argc, const char** argv) {
   Verilated::commandArgs(argc, argv);
 }
 
-void
-verilated_command_args_add(int argc, const char** argv) {
-  Verilated::commandArgsAdd(argc, argv);
-}
-
```
Comment (djg) [inline verilated/src/verilated_shim.cpp:111]: Why did you remove this function?

File: verilated/src/vcd.rs [hunk 2]
```diff
@@ -24,10 +23,17 @@ mod ffi {
...
 
+#[cfg(unix)]
 fn cstr(path: &Path) -> io::Result<CString> {
+    use std::os::unix::ffi::OsStrExt;
     Ok(CString::new(path.as_os_str().as_bytes())?)
 }
 
+#[cfg(not(unix))]
+fn cstr(path: &Path) -> io::Result<CString> {
+    Ok(CString::new(path.to_str().expect("path is not valid utf-8").as_bytes())?)
```
Comment (djg) [inline verilated/src/vcd.rs:34]: Why panic here if the string isn't valid utf-8, when it returns `io::Result`?

Comment (xobs) [reply verilated/src/vcd.rs]: The quick answer is that I wanted an easy fix to try and get things building, because I wasn't sure how much work it would take.  It turns out that it wasn't much!

I've updated it now to return an `io::Error::InvalidData` if you pass it a path that cannot be used.

**Comment by xobs** [comment]
Also, I see there was a question about why I removed `verilated_command_args_add()`, but I can't see it now.  I removed it because it wasn't present in my version of verilator.  It looks like things got moved around in https://github.com/verilator/verilator/commit/84335b958015c2b15266cfbcc1b0a87b2d035c30 and as a result it doesn't actually get linked in.

**Comment by xobs** [comment]
Specifically, with that function call still in place, I get the following error:

```
[11:09:06 AM] D:/Code/verilated-rs> cargo run
   Compiling verilated v0.1.1 (D:\Code\verilated-rs\verilated)
   Compiling example v0.0.0 (D:\Code\verilated-rs\example)
error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\ [...] "msvcrt.lib"
  = note: libverilated-5150819e2e36f022.rlib(verilated_shim.o) : error LNK2019: unresolved external symbol "public: stat
ic void __cdecl Verilated::commandArgsAdd(int,char const * *)" (?commandArgsAdd@Verilated@@SAXHPEAPEBD@Z) referenced in
function verilated_command_args_add
          D:\Code\verilated-rs\target\debug\deps\example.exe : fatal error LNK1120: 1 unresolved externals


error: aborting due to previous error

error: could not compile `example`.

To learn more, run the command again with --verbose.
[11:09:19 AM] D:/Code/verilated-rs>
```
